UnityProject/Assets/Scripts/Factories/ItemFactory.cs

```diff
@@ -60,4 +70,19 @@ public GameObject SpawnIDCard(IDCardType idCardType, JobType jobType, List<Acces
 		ID.AddAccessList(allowedAccess);
 		return idObj;
 	}
+
+	/// <summary>
+	/// For spawning Meals/Food items. Pass the prefab from the crafting manager here to be spawned
+	/// Server Side only!
+	/// </summary>
+	public GameObject SpawnMeal(GameObject mealPrefab, Vector3 position){
```

Comment:
So this could in theory be expanded to more meals?

Reply:
Yes, just add the MealPrefab to recipes in crafting manager and then you can spawn it via ItemFactory.SpawnMeal(prefab, position). Serverside of course